torch/fx/experimental/migrate_gradual_types/constraint_generator.py

```diff
@@ -33,31 +35,244 @@ def generate_flatten_constraints(start_dim, end_dim, input, flattened, n, counte
     return Conj([c1, c2, *nat_constraints]), counter
 
 
-@register_inference_rule(torch.add)
-@register_inference_rule(operator.add)
-def add_inference_rule(n: Node, symbols, constraints, counter):
+# TODO
+@register_inference_rule("long")
+def long_inference_rule(n: Node, symbols, constraints, counter):
+    """
+    """
+    return [], counter
```

Comment:
What does `[]` imply here? Why are there so many rules following this same empty pattern with no docstring?

A comment explaining how to read these rules (at the top) would be helpful.

If you are going to have a long list of rules with the same body, perhaps auto-generate them in a loop.

Reply:
These are the list of rules we still need to do. I left them empty and I plan to write them in the next PRs. `[]` implies no constraint.